File: Records/21CYS/CB.EN.U4CYS21089/C++ and python/C++/STLs/stack.cpp
#include <stack>
#include <bits/stdc++.h>
#include <iterator>
using namespace std;
int main() {

    stack<string> s;
    s.push("Y");
    s.push("A");
    s.push("S");
    s.push("W");
    s.push("A");
    s.push("N");
    s.push("T");
    s.push("H");

    cout << "size of stack: " << s.size() << endl;

    while (!s.empty()) {
        cout << s.top() << " ";
        s.pop();
    }
    cout << endl;
    cout << "size of stack: " << s.size() << endl;
}

File: Records/21CYS/CB.EN.U4CYS21089/C++ and python/Python/ADTs/stack.py
class stack1():
    def __init__(self, mx):
        self.qwe = []
        self.size = 0
        self.max_size = mx

    def push(self, value):
        if (len(self.qwe) < self.max_size):
            self.size += 1
            self.qwe.append(value)
        else:
            print("The stack is full")

    def pop(self):
        if len(self.qwe) > 0:
            self.qwe.pop()
        else:
            print("Stack empty")

    def top(self):
        print("The element on top: ", self.qwe[len(self.qwe)-1])

    def size(self):
        print(len(self.qwe))

    def isEmpty(self):
        if self.qwe:
            pass
        else:
            print("The stack is empty")

    def output(self):
        print(self.qwe)


v = stack1(3)
v.push("sbcsj")
v.push("ksjbck")
v.push("iscksd")
v.push("schduk")
v.top()
v.output()
